pkg/helm/actions/get_repos.go

```diff
@@ -0,0 +1,113 @@
+package actions
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+	"net/http"
+
+	"helm.sh/helm/v3/pkg/repo"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	HelmChartRepositoryGVK = schema.GroupVersionResource{
```

Comment:
Do we need this var as an exported one? Similarly for functions in this package?

Reply:
This fields and functions not necessarily be exported, Fixed this.